lib/net/ssh/connection/channel.rb

```diff
@@ -274,7 +274,7 @@ def wait
     # invoked, but the server has not yet responded with a CHANNEL_CLOSE
     # packet of its own.
     def closing?
-      @closing
+      @closing and @sent_close
```

Comment:
is this neccesary? only place we seems to be using this is https://github.com/net-ssh/net-ssh/blob/master/lib/net/ssh/connection/session.rb#L223

maybe we should change that ```if closing?``` to ```closed?``` and implement ```closed``` as you've described.